createMedicCWBenchmark.sh

```diff
@@ -32,10 +32,14 @@ git push origin medic-cw-benchmarks
 # delete older benchmark
 cd benchmarks/setup/
 
-nameOfOldestBenchmark=$(ls | grep medic-cw- | sort | head -n 1)
-
-./stopBenchmarks.sh $nameOfOldestBenchmark
-./deleteBenchmark.sh $nameOfOldestBenchmark
+cw=$(date +%V)
+if [ $cw -gt 4 ]
+then
+  nameOfOldestBenchmark=$(ls | grep medic-cw- | sort | head -n 1)
+  ./deleteBenchmark.sh $nameOfOldestBenchmark
+else
+  echo "We currently not support to delete benchmarks, before calendar week 5. Our deletion logic is not sophisticated enough. Please delete the benchmark manually."
+fi
```

Comment:
🔧 This message might get lost in the output produced by the lines below. Let's move this echo all the way in the end as a clear warning

Reply:
Yeah I thought about that but I don't think that there is too much output afterwards I don't want to add another conditional logic etc. I added some markers and colors I think this helps

![highlight](https://user-images.githubusercontent.com/2758593/149408335-d87bc7ce-7524-49c1-ab7c-161ca2ecd83e.png)